apps/sft/deepseek_v3.yaml

```
@@ -0,0 +1,70 @@
+# >>> python -m apps.sft.main --config apps/sft/deepseek_v3.yaml
+
+
+# TODO: required by torchtitan
+# https://github.com/pytorch/torchtitan/blob/2f1c814da071cc8ad165d00be6f9c1a66f8e1cce/torchtitan/distributed/utils.py#L265
+comm:
+  trace_buf_size: 0
+
+model_name: "deepseek-ai/DeepSeek-V3.1-Base"
+
+model:
+  name: deepseek_v3
+  flavor: 16B
+  hf_assets_path: hf://${model_name}
+
+processes:
+  procs: 8
+  with_gpus: true
+
+optimizer:
+  name: AdamW
+  lr: 1e-5
+  eps: 1e-8
+
+lr_scheduler:
+  warmup_steps: 200
+
+training:
+  local_batch_size: 1
+  seq_len: 2048
+  max_norm: 1.0
+  steps: 1000
+  compile: false
+  dataset: "c4"
+
+parallelism:
+  data_parallel_replicate_degree: 1
+  data_parallel_shard_degree: -1
+  tensor_parallel_degree: 1
```

Based on a conversation with @tianyu-l this should work for 32 nodes
```suggestion
  data_parallel_shard_degree: -1
  tensor_parallel_degree: 8
```

It also may be worth looking at some of the runs [here](https://verl.readthedocs.io/en/latest/perf/dpsk.html#deepseek-671b) for reference.

Not sure of the latest status of ETP in titan but it'd be worth confirming that as well.